### PR TITLE
Release blog post 0.113

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Small improvement to the 0.113 release blog post.

- Add chapter about the new sub-second precision of delays.
- Replaced commits, in the new integrations and platforms sections with a more humanized version.
- Deduplicated the new platforms second, as it contained new platforms of new integrations. 
- A small tweak in the beta release header.
- Updated all changes
- Added in new breaking change from beta (ZHA)
- Added secret news

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by zsarnett — DISMISSED** [review]

**Comment by frenck** [comment]
hehe thanks @zsarnett, but I wasn't done yet, should have marked it as Draft. Sorry 🙏

**Review by ludeeus — DISMISSED** [review]

**Review by balloob — DISMISSED** [review]

File: source/_posts/2020-07-01-release-113.markdown [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
-title: "0.113: Beta release notes"
-description: "0.113: Beta release notes"
-date: 2020-07-14 00:00:00
+title: "0.113: Automations & Scripts, and even more performance!"
+description: "0.113: Great new features to our Automations & Scripts, and even more performance improvements!"
```
Comment (balloob) [inline source/_posts/2020-07-01-release-113.markdown]: Could we try to not just reword the title but give some examples?

Comment (balloob) [reply source/_posts/2020-07-01-release-113.markdown]: Maybe: Run scripts in parallel, switch statements for automations and sub-second precision on waits.

Comment (frenck) [reply source/_posts/2020-07-01-release-113.markdown]: Done

**Pull request merged**